src/components/Global/AlertToast.vue

```diff
@@ -23,7 +23,7 @@ watch(() => alertStore.alertText, () => {showAlert.value = true})
     position: absolute;
     bottom: 0;
     left: 0;
-    z-index: 1000;
+    z-index: 9999;
```

Comment:
here's a hot tip: avoid not keeping track of your z-indexes. Only use 9999 for something you absolutely know that has to go on top of everything no matter what. Otherwise, I would suggest starting from z-index: 2 and going up from there. It's easier to do it now than later

Reply:
ahhh thats a good practice, I can change this one to 2401, the v-overlay component which was blocking it sits at 2400